MFVNeutralino/test/K0Tracks/bkgsub.py

```diff
@@ -9,17 +9,18 @@
 out_fn = os.path.join(batch, in_bn)
 sample = in_bn.replace('.root', '')
 
-ps = plot_saver(plot_dir('k0bkgsub_%s/%s' % (batch, sample)), size=(600,600))
+ps = plot_saver(plot_dir('v0bkgsub_%s/%s' % (batch, sample)), size=(600,600))
```

Comment:
@duquale ust curious, why V0 instead of K0? Usually V0 (in other CMS contexts) would refer to K0 and Λ0, but the latter is heavier than the mass range you're focusing on.